src/arrayMethodReduce.js

```diff
@@ -4,8 +4,15 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(callback, ...initialValue) {
```

Comment:
Don't use '...initialValue' as it can take multiple arguments, but according to the native 'reduce' method, it should take only one argument after the callback.